scalableminds/webknossos
# Improve error reporting for task creation + mesh loading

### Steps to test:
- Try to create task for non-existing project (e.g. in bulk task creation)
- Try to load mesh for segment that is not present in the mesh file
- Both should have readable error messages

fixes #5525
